src/Packetery/Module/Order/PacketAutoSubmitter.php

```diff
@@ -100,12 +100,13 @@ function ( int $orderId ) {
 	/**
 	 * Handle event.
 	 *
-	 * @param string $event   Event.
-	 * @param int    $orderId WC Order.
+	 * @param string    $event               Event.
+	 * @param int       $orderId             WC Order.
+	 * @param bool|null $triggeredByFrontend Tells if event is triggered by frontend logic. NULL is passed when data from previous versions are being processed.
 	 *
 	 * @return void
 	 */
-	public function handleEvent( string $event, int $orderId ): void {
+	public function handleEvent( string $event, int $orderId, ?bool $triggeredByFrontend = null ): void {
```

Comment:
Výchozí hodnota false by umožnila udělat přehlednější podmínku pro metodu shouldUpdateOrderStatus

Reply:
Nesouhlasím. NULL reprezentuje volání ze staré verze pluginu, kde se změna stavu neprováděla. Je to napsáno v phpDocu metody. Výchozí hodnota false by říkala, že volání přišlo z administrace. To by ale nemusela být pravda.